.github/workflows/reusable-build-test-release.yml

```diff
@@ -1650,6 +1830,7 @@ jobs:
     if: always() && needs.pre-publish.result == 'success' && github.event_name != 'pull_request' && github.event_name != 'schedule'
     needs:
       - pre-publish
+      - run-escu-tests
```

Comment:
It should be probably in `pre-publish` `needs`

Reply:
@uoboda-splunk For now, we don't want to mark ESCU tests as publish blocker. Since this is a new test type and dev teams are not much familiar with it we don't want to block publish based on the results of ESCU tests. So I haven’t added this in pre-publish. I have created a Jira for the same https://splunk.atlassian.net/browse/AQA-627.

But we have a requirement to publish the result of ESCU in Github Releases. So publish stage should run after _run-escu-tests_ stage. Because of that, I have added _run-escu-tests_ in the needs of publish stage.

Reply:
Ok